LDMP/utils.py

```diff
@@ -0,0 +1,85 @@
+import importlib
+import tempfile
+import typing
+from pathlib import Path
+
+import qgis.core
+from osgeo import gdal
+from openpyxl.drawing.image import Image
```

Comment:
This throws an error on initial loading the plugin as `utils` is imported from at the beginning __init__.py, but `openpyxl` is potentially not available until later, after `ext-libs` is loaded. This can be fixed by moving `from openpyxl.drawing.image import Image` to be within the `maybe_add_image_to_sheet` function in `__init,py__`. Likely this only comes up on Windows or other systems where QGIS is using its own ython install that users haven't modified by adding any further dependencies (like openpyxl)

Reply:
@azvoleff I couldn't reproduce the error when installing the plugin on my window OS, could you give the steps to get the error and/or screenshot of the error, @ricardogsilva has opened the issue here https://github.com/ConservationInternational/trends.earth/issues/490, thanks.